src/main/resources/view/DarkTheme.css

```diff
@@ -337,12 +337,12 @@
     -fx-background-radius: 0;
 }
 
-#tags {
+#userType {
     -fx-hgap: 7;
     -fx-vgap: 3;
 }
 
-#tags .label {
+#userType .label {
```

Comment:
Accidental edit?

Reply:
yup it's correct, because the `FXML` element in `PersonCard` has been renamed from `tags` to `userType`, when I didn't change the element name in `DarkTheme.css`, no styles were applied to the `userType` element in the Ui!